src/backend/core/models.py

```diff
@@ -325,3 +337,47 @@ def clean_fields(self, exclude=None):
         else:
             raise ValidationError({"name": f'Room name "{self.name:s}" is reserved.'})
         super().clean_fields(exclude=exclude)
+
+    def start_recording(self):
+        """Create a new related recording object to which Livekit will be able to save a file."""
+        return Recording.objects.create(room=self)
```

Comment:
the need to enforce a single recording session per room at a time, might be declared somewhere else in the code, I am only starting the review